README.md

```diff
@@ -29,21 +29,40 @@ To try PyScript, import the appropriate pyscript files into the `<head>` tag of
         src="https://pyscript.net/releases/2023.11.1/core.js"
     ></script>
 </head>
+<body>
+    <script type="py" terminal>
+        from pyscript import display
+        display("Hello World!") # this goes to the DOM
+        print("Hello terminal") # this goes to the terminal
+    </script>
+</body>
 ```
 
-You can then use PyScript components in your html page. PyScript currently implements the following elements:
+You can then use PyScript components in your html page. PyScript currently offers various ways of running Python code:
 
--   `<py-script>`: can be used to define python code that is executable within the web page. The element itself is not rendered to the page and is only used to add logic
--   `<py-repl>`: creates a REPL component that is rendered to the page as a code editor and allows users to write executable code
+-   `<script type="py">`: can be used to define python code that is executable within the web page.
+-   `<script type="py" src="hello.py">`: same as above, but the python source is fetched from the given URL.
+-   `<script type="py" terminal>`: same as above, but also creates a terminal where to display stdout and stderr (e.g., the output of `print()`); `input()` does not work.
+-   `<script type="py" terminal worker>`: run Python inside a web worker: the terminal if fully functional and `input()` works.
```

Comment:
Would you guys mind adding a reason (or pointer to one) about why a user would choose to run python in the worker instead of as main ?